xbmc/windowing/gbm/DRMUtils.cpp

```diff
@@ -661,7 +683,12 @@ void CDRMUtils::DestroyDrm()
 {
   RestoreOriginalMode();
 
-  drmDropMaster(m_fd);
+  auto ret = drmDropMaster(m_fd);
```

Comment:
shouldn't this be done only if we are master?

Reply:
It's safe to call even if we aren't master